pkgs/buildCatppuccinPort/package.nix

```
@@ -17,20 +17,21 @@ stdenvNoCC.mkDerivation (
 
   args'
   // {
-    version = args'.version or (builtins.substring 0 7 finalAttrs.src.rev);
+    pname = args'.pname or "ctp-${args'.port}";
```

I'd personally prefer to be more verbose with this.
```suggestion
    pname = args'.pname or "catppuccin-${args'.port}";
```